### DISCUSSION TITLE
Actions log: Negative duration

### DISCUSSION THREAD

**Discussion by peaceiris** [community/community#12674] [Actions] [answered]
https://github.com/peaceiris/hugo-extended-docker/runs/5504472261?check_suite_focus=true

![スクリーンショット 2022-03-11 9 23 02](https://user-images.githubusercontent.com/30958501/157777833-645afe52-50dc-4e15-a621-a99e7fe8b913.png)

Is this an expected behavior?

**Comment by ethomson** [comment, marked as answer]
Hi @peaceiris - thanks for letting us know, and apologies for this problem. There was a brief bug in workflow results that showed a `-1s` time for some steps. I hope that you're no longer seeing this, but please let me know if it persists.

**Reply by peaceiris** [reply]
It seems to have been fixed. Thank you for the quick patch!
